### PR TITLE
:bug: fixes for:

### PR DESCRIPTION
**Pull request by zimny**
:white_check_mark: profile property not properly passed in the CLI for mira cicd
:white_check_mark: removed unused stack name from the CLI
:white_check_mark: removed useless console logs
:white_check_mark: added mira docs into CLI
:white_check_mark: fixed requirement for app property being specified in the config file for mira docs
:white_check_mark: update AWS CDK dependency to 1.44.0
:white_check_mark: updated docs

### CODE REVIEW

**Comment by zimny** [comment]
Solves #7, #2, #1 

#1 was addressed with CDK 1.44.0 which is not the latest one, but there's other dependency to auto-delete-bucket which is coupled with 1.44.0.

File: docs/quick-start/README.md [hunk 1]
```diff
@@ -59,13 +85,60 @@ At this step you should have your development environment deployed and ready to
...
+           ```
+
+This list is dynamic, so you can add or remove target environments.
 
 
 ### Permissions for CodeBuild
 
+AWS CodeBuild requires access to target environments in order to deploy the application with the CI.
+To achieve it, Mira expects dedicated service role to be deployed first.
+See [cicd](../cicd/README.md) for more information about the `Deploy Role`.
```
Comment (irelandm) [inline docs/quick-start/README.md:140]:
```suggestion
See [CICD](../cicd/README.md) for more information about the `Deploy Role`.
```

File: docs/quick-start/README.md [hunk 2]
```diff
@@ -59,13 +85,60 @@ At this step you should have your development environment deployed and ready to
...
+            {
+             "name": "Staging",
+             "env": {
+               "account": "YOUR_NUMBER",
+               "region": "YOUR_REGION"
+             },
+             "profile": "YOUR_PROFILE_NAME",
+             "withDomain": false
+           }]
+           ```
```
Comment (irelandm) [inline docs/quick-start/README.md:131]: These backticks need to be at the start of the line

File: docs/cicd/README.md [hunk 3]
```diff
@@ -1,7 +1,32 @@
...
+
+Mira utilize cloud native services for delivery pipeline. 
+
+Key AWS Services used:
+* [AWS CodePipeline](https://aws.amazon.com/codepipeline/getting-started/)
+* [AWS CodeBuild](https://docs.aws.amazon.com/codebuild/latest/userguide/getting-started.html)
+* [AWS CodeCommit](https://docs.aws.amazon.com/codecommit/latest/userguide/getting-started-cc.html)
+
+AWS CodePipeline is used for overall orchestration of the delivery pipeline. It is capable of detecting changes in the
+CodeCommit repository and trigger itself. Single CodePipeline is defined by the pipeline that consists of 1 or more stages.
```
Comment (nigelhanlon) [inline docs/cicd/README.md:13]:
```suggestion
CodeCommit repository and triggers when required. A single CodePipeline is defined for delivery, with the pipeline itself built from one or more stages of pre-configured actions.
```

File: docs/cicd/README.md [hunk 4]
```diff
@@ -1,7 +1,32 @@
...
+
+For code deployment, Mira uses AWS CodeBuild which a dedicated service for artifacts build up and optionally deployment.
+Behind the scenes CodeBuild triggers the same Mira commands as a Developer from local workstation.
+In order to provide permissions for CodeBuild to deploy stacks, Mira provision dedicated Role(s) in predefined accounts.
+
+Mira CI/CD can deploy the code the the same AWS account where it is hosted, but also to different accounts.
```
Comment (nigelhanlon) [inline docs/cicd/README.md:19]:
```suggestion
Mira CI/CD can deploy the code into the same AWS account where it is hosted, but also to different accounts as required.
```

File: docs/cicd/README.md [hunk 5]
```diff
@@ -1,7 +1,32 @@
...
+You define deployment stages by configuring the `cicd.accounts` value in the `config/default.json` file.
+
+Typical pipeline looks in the following way:
```
Comment (nigelhanlon) [inline docs/cicd/README.md:22]:
```suggestion
A typical pipeline would look like this:
```

**Pull request merged**